pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -171,13 +171,17 @@ func (i *CommandInitOption) Complete() error {
 	}
 
 	// Determine whether KarmadaDataPath exists, if so, delete it
+	// Related issue: https://github.com/karmada-io/karmada/pull/1455
```

Comment:
we don't need to explain it in code. It can be recorded by `git commit`

Reply:
fixed

Reply:
Yes, that's exactly what I want to say. We don't need to describe the history of the code, we just need to explain why we do it.